src/platform/julius.c

```diff
@@ -325,9 +326,11 @@ static void teardown(void)
 
 int main(int argc, char **argv)
 {
+    SDL_Log("Built with git commit: %s\n", GIT_COMMIT_HASH);
+    
     const char *custom_data_dir = (argc > 1 && argv[1]) ? argv[1] : NULL;
     setup(custom_data_dir);
-
+    
```

Comment:
Picky picky! Get rid of this extra space :P

Reply:
removed newline